### PR TITLE
Fix WS issues with CF Benchmarks adapter

### PR DESCRIPTION
**Pull request by cds95**
## Description

 - Address failing HTTP requests to CF Benchmarks
 - Fix bug where sending requests for different feeds always returned the price of the first one.  This issue was caused due to the WS `subscriptionKey` always being the same as the `id` field in the WS subscription message was always being ignored.  We should come up with a better solution for this in the future.  I've created a ticket for this here https://app.shortcut.com/chainlinklabs/story/24721/gracefully-handle-cases-where-ws-subscription-payload-messages-contains-ignored-key

## Changes

- Update adapter HTTP endpoint
- Add a `cacheID` field to ensure different feeds use different keys

## Steps to Test

Make sure you enable websockets before testing.  

1. Send this payload and get a successful response.  Additionally verify that websockets is running from the adapter logs.

```
{
  "id": "1",
  "data": {
    "index": "ETHUSD_RTI"
  }
}
```
2. Send below payload and verify response
```
{
  "id": "1",
  "data": {
    "index": "BRTI"
  }
}
```

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/cfbenchmarks/src/adapter.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const makeWSHandler = (config?: Config): MakeWSHandler => {
   }
   const getSubscription = (type: 'subscribe' | 'unsubscribe', id?: string) => {
     if (!id) return
-    return { type, id, stream: 'value' }
+    return { type, id, stream: 'value', cacheID: id } // Temporarily add another key named cacheID that won't be ignored when generating the subscription key in getSubsID
```
Comment (cds95) [inline packages/sources/cfbenchmarks/src/adapter.ts:32]: I've created this ticket to handle this better in the future. https://app.shortcut.com/chainlinklabs/story/24721/gracefully-handle-cases-where-ws-subscription-payload-messages-contains-ignored-key.

**Review by austinborn — DISMISSED** [review]

File: packages/sources/cfbenchmarks/src/adapter.ts (hunk 1 above)
Comment (austinborn) [reply packages/sources/cfbenchmarks/src/adapter.ts:32]: Have you checked if this would affect any other adapters?

Comment (austinborn) [reply packages/sources/cfbenchmarks/src/adapter.ts:32]: It looks like every other definition of `getSubscription` for EAs uses a single asset pair, so I'm guessing this is unique to `cfbenchmarks`.

Comment (cds95) [reply packages/sources/cfbenchmarks/src/adapter.ts:32]: yup it's for `cfbenchmarks` only.  It doesn't affect other adapters.

**Review by boxhock — DISMISSED** [review]

**Review by alejoberardino — DISMISSED** [review]

**Review by justinkaseman — APPROVED** [review]

**Pull request merged**